ios/FluentUI/Vnext/Persona/PersonaViewTokens.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+class MSFPersonaViewTokens: MSFCellBaseTokens {
+    override init() {
+        super.init()
+
+        self.themeAware = true
+        updateForCurrentTheme()
+    }
+
+    @objc open func didChangeAppearanceProxy() {
+        updateForCurrentTheme()
+    }
+
+    override func updateForCurrentTheme() {
+        let currentTheme = theme
+        let personaViewAP = currentTheme.MSFPersonaViewTokens
+        let listCellAP = currentTheme.MSFListCellTokens
+
+        borderColor = listCellAP.borderColor
+        disclosureIconForegroundColor = listCellAP.disclosureIconForegroundColor
+        labelColor = listCellAP.labelColor
+        leadingViewColor = listCellAP.leadingViewColor
+        sublabelColor = personaViewAP.sublabelColor
+        trailingItemForegroundColor = listCellAP.trailingItemForegroundColor
+
+        backgroundColor = listCellAP.backgroundColor.rest
+        highlightedBackgroundColor = listCellAP.backgroundColor.pressed
+
+        cellHeightOneLine = listCellAP.cellHeight.oneLine
```

Comment:
All these assignments could be inherited by the base class. You can override the updateForCurrentTheme() and do the additional PersonaView related ones.

Reply:
We just talked on our standup meeting and the fact that sgen creates extensions for both classes (which will clash), we can just leave it as is now and work on a potentially better solution after this PR is merged.